test/cql-pytest/test_materialized_view.py

```diff
@@ -1189,3 +1192,63 @@ def test_base_partition_deletion_with_low_timestamp(cql, test_keyspace):
             # Verify we get only the row with the newer timestamp
             assert [2] == [x.c for x in cql.execute(f"SELECT c FROM {table} WHERE p=0")]
             assert [2] == [x.c for x in cql.execute(f"SELECT c FROM {mv} WHERE p=0")]
+
+# Verify that creating a materialized view with an ID that is already used fails.
+def test_creating_mv_with_existing_id(cql, test_keyspace):
+    # Returns the ID of a given CF. `cf_full_name` is of form `ks_name.cf_name`.
```

Comment:
All this explanation about "full name" (in the comment and in the parameter name) isn't important any more - the new code (using DESC MATERIALIZED VIEW on a single view) could work on a non-full name, if you used "USE".
Also, since you're planning to pass a view, not a table, and (see my next comment) use DESC MARTERIALIZED VIEW), I think it's better to call the parameter "view", not "cf_full_name".